index.html

```diff
@@ -68,9 +68,9 @@ <h2><a>Documentation</a></h2>
 
 <h2><a>Getting started</a></h2>
 
-<p>The recommended way of installing EasyBuild is via the
-<a href="http://easybuild.readthedocs.io/en/latest/Installation.html#bootstrapping-procedure">documented bootstrap
-procedure</a>. You should <a href="http://easybuild.readthedocs.io/en/latest/Configuration.html">configure</a>
+<p>The recommended way of installing EasyBuild is via 
+<a href="https://docs.easybuild.io/en/latest/Installation.html#using-pip-to-install-easybuild">pip</a>. 
+You should <a href="http://easybuild.readthedocs.io/en/latest/Configuration.html">configure</a>
```

Comment:
@verdurin Let's make this link `https://docs.easybuild.io` too?

http://docs.easybuild.io/en/latest/Configuration.html